src/contracts/scripts/SafeExample.s.sol

```diff
@@ -30,12 +32,26 @@ contract SafeExample is ArbScript {
     }
 
     function payload0002() public setUp broadcasted(safe) {
-        USDC.approve(kreskoAddr, 1);
-        USDC.approve(kreskoAddr, 0);
-        kresko.setSingleSwapRouteSCDP(SwapRouteSetter({assetIn: krETHAddr, assetOut: kissAddr, enabled: false}));
-        IExtendedDiamondCutFacet(kreskoAddr).executeInitializer(
-            deployPayload(type(ExamplePayload0002).creationCode, "", 2),
-            abi.encodeWithSelector(ExamplePayload0002.executePayload.selector)
-        );
+        bytes32[] memory tickers = new bytes32[](4);
+        tickers[0] = "ETH";
+        tickers[1] = "BTC";
+        tickers[2] = "USDC";
+        tickers[3] = "ARB";
+
+        uint256[] memory staleTimes = new uint256[](4);
+        staleTimes[0] = 30;
+        staleTimes[1] = 30;
+        staleTimes[2] = 30;
+        staleTimes[3] = 30;
+
+        bool[] memory invertPyth = new bool[](4);
+
+        ICommonConfigFacet.PythConfig memory pythConfig = ICommonConfigFacet.PythConfig({
+            pythIds: tickers,
```

Comment:
It would be best if we just create configuration once in the script and test with that

eg. state and private function for config in script:
```
    bytes32[] internal tickers = [bytes32("ETH"), "BTC", "USDC", "ARB"];

    function getPythConfig() private view returns (ICommonConfigFacet.PythConfig memory pythConfig) {
        pythConfig.pythIds = new bytes32[](tickers.length);
        pythConfig.invertPyth = new bool[](tickers.length);
        pythConfig.staleTimes = new uint256[](tickers.length);
        for (uint256 i; i < tickers.length; i++) {
            Oracle memory config = kresko.getOracleOfTicker(tickers[i], Enums.OracleType.Pyth);
            pythConfig.pythIds[i] = config.pythId;
            pythConfig.invertPyth[i] = config.invertPyth;
            pythConfig.staleTimes[i] = 30;
        }
    }
```

Reply:
in `getPythConfig` we also fix the issue of pythIds as we simply copy everything that exists on chain and just replace the staleTime. 

pythId !== ticker: 

ticker is eg `bytes32("ETH")` but the pythId for this ticker is `0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace` which can be found here https://pyth.network/developers/price-feed-ids or in `pyth_stable_ids.json` in the repo